lua/dap-go.lua

```diff
@@ -59,7 +59,7 @@ local function setup_adapter()
     local port = configuration.port or '38697'
     local addr = string.format('%s:%s', host, port)
 
-    if configuration.options.env then
+    if configuration.options and configuration.options.env then
```

Comment:
`configurations.options` was not present in my case and caused the common field not found when trying to access `env`.

Reply:
lgtm